kubetest/extract.go

```diff
@@ -45,6 +45,7 @@ const (
 	version             // v1.5.0, v1.5.0-beta.2
 	gcs                 // gs://bucket/prefix/v1.6.0-alpha.0
 	load                // Load a --save cluster
+	bazel               // A pre/postsubmit bazel build version, prefixed with bazel/
```

Comment:
The tool that created the version does not seem germane to the format of the version string. Can we choose a better abstraction for this type of version string?